docker-compose.yml

```diff
@@ -6,29 +6,32 @@ services:
       - ./certs:/certs
 
   tinkerbell:
-    build: .
-    environment: 
+    image: quay.io/packet/tinkerbell:workflow
+    environment:
       FACILITY: ${FACILITY:-lab1}
       PACKET_ENV: ${PACKET_ENV:-testing}
       PACKET_VERSION: ${PACKET_VERSION:-5efab5ef3a42cb88f2d54f4ed3201c2dd6797b7d}
-      ROLLBAR_TOKEN: ${ROLLBAR_TOKEN:-9b78d0ad01d1467aa92c49c3a349b79d}
-      ROLLBAR_DISABLE: ${ROLLBAR_DISABLE:-0}
-      MAX_WORKFLOW_DATA_VERSIONS: 5
+      ROLLBAR_TOKEN: ${ROLLBAR_TOKEN:-##########################}
+      ROLLBAR_DISABLE: ${ROLLBAR_DISABLE:-1}
       PGDATABASE: tinkerbell
       PGHOST: db
       PGPASSWORD: tinkerbell
       PGPORT: 5432
       PGSSLMODE: disable
       PGUSER: tinkerbell
     depends_on:
-      - "certs"
-      - "db"
+      certs:
+        condition: service_started
+      fluentbit:
+        condition: service_started
+      db:
+        condition: service_healthy
     healthcheck:
       test: ["CMD-SHELL", "wget -qO- 127.0.0.1:42114/cert"]
       interval: 5s
       timeout: 2s
       retries: 30
-    volumes: 
+    volumes:
       - ./certs:/certs/${FACILITY}
```

Comment:
I believe Tinkerbell should be independent of `facility`. Though it would require change across the repo and can be a separate PR.
@nathangoulding @mmlb What do you think?